lib/debug/asserts.js

```diff
@@ -43,8 +43,23 @@ shaka.asserts.notImplemented = function() {};
 shaka.asserts.unreachable = function() {};
 
 
+shaka.asserts.patchAssert_ = function() {
+  var assert = console['assert'];
```

Comment:
I think this doesn't need to be quoted.  Dot notation should work fine, since closure will not rename browser-built-ins and thingThatExists.nonExistentMember will be undefined, but not cause an exception.  Please correct me if I missed something.